assets/src/design-system/components/contextMenu/menuItem.js

```diff
@@ -141,6 +142,7 @@ export const MenuItem = ({
         disabled={disabled}
         onClick={handleClick}
         onFocus={onFocus}
+        role="menuitem"
```

Comment:
This is causing a js test to fail: 
<img width="931" alt="Screen Shot 2021-05-18 at 4 07 56 PM" src="https://user-images.githubusercontent.com/10720454/118734506-638c3100-b7f3-11eb-955e-49d5f0c20584.png">